### ISSUE TITLE
Cop to detect redundant `# frozen_string_literal: true` magic comment

### ISSUE THREAD

**Issue by yahonda** [rubocop/rubocop#11043]
## Is your feature request related to a problem? Please describe.
I'd like to know if there are any cop that fixes duplicate / redundant `# frozen_string_literal: true` magic comments. Refer to https://github.com/rails/rails/pull/46183 for the background.

## Describe the solution you'd like

* Bad
```ruby
# frozen_string_literal: true

# frozen_string_literal: true

def foo; end
```

* Good
```ruby
# frozen_string_literal: true

def foo; end
```

## Describe alternatives you've considered
Find each file manually to see if there are redundant `# frozen_string_literal: true` in each *.rb file.

**Comment by koic** [comment]
Yeah, I also saw that PR https://github.com/rails/rails/pull/46183 and am just starting to implement it. And I'm considering including a slightly more comprehensive feature. Thank you ❤️
